File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Concert/ConcertResultLocationDto.cs
using System.ComponentModel.DataAnnotations;

namespace ArenaGestor.APIContracts.Concert
{
    public class ConcertResultLocationDto
    {
        [Required]
        public int LocationId { get; set; }

        [MaxLength(50)]
        [Required]
        public string Place { get; set; }

        [Required]
        public ConcertResultCountryDto Country { get; set; }

        [MaxLength(500)]
        [Required]
        public string Street { get; set; }

        [Required]
        public int Number { get; set; }
    }
}

File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Concerts/ConcertResultLocationDto.ts
import { ConcertResultCountryDto } from "./ConcertResultCountryDto";

export class ConcertResultLocationDto {
    locationId: Number = 0;
    place: String = "";
    street: String = "";
    number: Number = 0;
    country: ConcertResultCountryDto = new ConcertResultCountryDto();
}